src/randomInt.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generate an inclusive random integer between `from` and `to`.
+ *
+ * @param from - The minimum value.
+ * @param to - The maximum value.
+ * @returns The random integer.
+ * @signature
+ *   R.randomInt(from, to)
+ * @example
+ *   R.randomInt(1, 10) // => 5
+ *   R.randomInt(1n, 10n) // => 7n
+ * @dataFirst
+ * @category Number
+ */
+export function randomInt<T extends bigint | number>(
+  from: T,
+  to: Widen<T>,
+): Widen<T> {
+  if (to <= from) {
```

Comment:
Because our range is inclusive of the edges, it's okay to call this function with the same value for `to` and `from` (`randomInt(2, 2)`), as it would result in a consistent value which makes sense. This might happen when the numbers are not fixed and are generated by other calculations.
```suggestion
  if (to < from) {
```

Reply:
Done